### ISSUE TITLE
images are too big

### ISSUE THREAD

**Issue by daguilarc** [sfyimby/sfyimby#24]
also when posting to facebook, it doesnt automatically pick the cover photo as the preview image.

**Comment by bufordsharkley** [comment]
This may have to do with tags we need to add to let Facebook pick it up. (I
did this to have Twitter work: "Twitter card")

Could you look up to see if this is a thing?

**Comment by daguilarc** [comment]
yeah here you go Mark: https://developers.facebook.com/docs/sharing/webmasters

**Comment by bufordsharkley** [comment]
Cool. I think I could add this. Maybe I'll knock it out soon.
